examples/MediaServices/scale_encoding_units.php

```diff
@@ -22,17 +22,14 @@
  *
  * @link      https://github.com/windowsazure/azure-sdk-for-php
  */
-require_once 'vendor/autoload.php';
+require_once __DIR__.'/../vendor/autoload.php';
 
 use WindowsAzure\Common\ServicesBuilder;
 use WindowsAzure\Common\Internal\MediaServicesSettings;
 use WindowsAzure\MediaServices\Models\EncodingReservedUnitType;
 
-// Settings
-date_default_timezone_set('America/Los_Angeles');
-
-$account = '<your media services account name>';
-$secret = '<your media services account key>';
+// read user settings from config
+include_once 'userconfig.php';
```

Comment:
The comment is intended to encourage the same expression (and behavior) for requiring files across the project:
https://secure.php.net/manual/en/function.include.php
>The include construct will emit a warning if it cannot find a file; this is different behavior from require, which will emit a fatal error.